### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,8 +53,11 @@ import { DrilldownCalendar } from '@manufac/reactjs-calendar-heatmap';
   color={color}
   overview={overview}
   response={response}
+  showDayXAxisLabels={showDayXAxisLabels}
```
Comment (maneetgoyal) [inline README.md:58]: Indentation

**Review by maneetgoyal — CHANGES_REQUESTED** [review]
Comments not resolved

File: README.md (hunk 1 above)
Comment (qur786) [reply README.md:58]: In VS code and notepad++ the indentation is not there, don't know why it is appearing above here.
image
image

Comment (qur786) [reply README.md:58]: I  exchanged the position of  color and data attribute, It solved the problem.

**Review by maneetgoyal — APPROVED** [review]

**Pull request merged**